openstates/ar/legislators.py

```diff
@@ -48,7 +48,13 @@ def scrape_member(self, chamber, term, member_url):
 
         party = name_and_party[-1]
 
-        if party == '(R)':
+        if ' '.join(name_and_party[1:]) == 'David Johnson' and "(" not in party:
+                full_name = ' '.join(name_and_party[1:])
+                party = 'Democratic'
+        elif ' '.join(name_and_party[1:]) == 'David Johnson' and "(" in party:
```

Comment:
If a party _does_ come back, then `' '.join(name_and_party[1:]) == 'David Johnson'` will be false, right?